docs/pages/management/guides/docker.mdx

```diff
@@ -7,13 +7,11 @@ h1: Run Teleport using Docker
 This guide will explain how to run a container using one of Teleport's Docker
 images and execute commands on that container via Teleport's `tsh` client.
 
-<Notice scope={["cloud"]} type="tip">
-
 Since all of Teleport's services are run from the same binary, you can
-use our Docker image to run Node services (e.g., the Database Service or App
-Service) or explore the Auth and Proxy Services locally.
-
-</Notice>
+use our Docker image to run Teleport services (e.g., the Database Service or App
+Service) or explore the Auth and Proxy Services locally. The distroless container
+does not allow running shells by default so an example of adding another Ubuntu container for
+server access is given.
```

Comment:
```suggestion
Service) or explore the Auth and Proxy Services locally. In this guide, we will also show you how to join a server (in this case, an Ubuntu container) to your local Dockerized Teleport cluster.
```
Rather than say, "Here's this cumbersome limit to our own product that we're overcoming in this guide", I think it would make sense to use a positive formulation. I don't think we lose any meaning this way.